diff --git a/src/context/ThemeContext.tsx b/src/context/ThemeContext.tsx
index 6ef0dd8..2530eff 100644
--- a/src/context/ThemeContext.tsx
+++ b/src/context/ThemeContext.tsx
@@ -21,7 +21,7 @@ export const ThemeProvider = ({ children }: { children: React.ReactNode }) => {
         if (window.matchMedia("(prefers-color-scheme: dark)").matches) {
             return "dark";
         }
-        return "light";
+        return "dark"; // Default to dark theme
     });
 
     useEffect(() => {
