diff --git a/packages/server/src/course/course.entity.ts b/packages/server/src/course/course.entity.ts
index 3b8689a6..1637f16a 100644
--- a/packages/server/src/course/course.entity.ts
+++ b/packages/server/src/course/course.entity.ts
@@ -2,6 +2,7 @@ import { Exclude } from 'class-transformer';
 import {
   BaseEntity,
   Column,
+  CreateDateColumn,
   DeleteDateColumn,
   Entity,
   JoinColumn,
@@ -101,6 +102,9 @@ export class CourseModel extends BaseEntity {
   @DeleteDateColumn()
   deletedAt?: Date;
 
+  @CreateDateColumn()
+  createDateColumn: Date;
+
   @OneToOne(
     () => OrganizationCourseModel,
     (organizationCourse) => organizationCourse.course,
diff --git a/packages/server/src/organization/organization.entity.ts b/packages/server/src/organization/organization.entity.ts
index fe026dca..7926d51c 100644
--- a/packages/server/src/organization/organization.entity.ts
+++ b/packages/server/src/organization/organization.entity.ts
@@ -2,6 +2,7 @@ import { Exclude } from 'class-transformer';
 import {
   BaseEntity,
   Column,
+  CreateDateColumn,
   Entity,
   JoinColumn,
   OneToMany,
@@ -114,4 +115,7 @@ export class OrganizationModel extends BaseEntity {
     (orgChatbotSettings) => orgChatbotSettings.organization,
   )
   userAuthStates: AuthStateModel;
+
+  @CreateDateColumn()
+  createDateColumn: Date;
 }
diff --git a/packages/server/src/queue/queue.entity.ts b/packages/server/src/queue/queue.entity.ts
index 5280d1de..7b296760 100644
--- a/packages/server/src/queue/queue.entity.ts
+++ b/packages/server/src/queue/queue.entity.ts
@@ -3,6 +3,7 @@ import { QuestionGroupModel } from '../question/question-group.entity';
 import {
   BaseEntity,
   Column,
+  CreateDateColumn,
   Entity,
   JoinColumn,
   ManyToOne,
@@ -102,4 +103,7 @@ export class QueueModel extends BaseEntity {
   @OneToMany(() => QueueChatsModel, (queueChat) => queueChat.queue)
   @Exclude()
   chats: QueueChatsModel[];
+
+  @CreateDateColumn()
+  createDateColumn: Date;
 }
