diff --git a/app/page.tsx b/app/page.tsx
index ffc0f3b..8ea4021 100644
--- a/app/page.tsx
+++ b/app/page.tsx
@@ -1,4 +1,5 @@
 import Features from "@/components/features";
+import Sponsors from "@/components/sponsors";
 // import Showcase from "@/components/showcase";
 
 import Hero from "@/components/hero";
@@ -8,6 +9,7 @@ export default function Home() {
     <div className="flex flex-col gap-12">
       <Hero />
       <Features />
+      <Sponsors />
       {/* <Showcase /> */}
     </div>
   );
