diff --git a/system-files/base/etc/greetd/config.toml b/system-files/base/etc/greetd/config.toml
index d291202..6027bb5 100644
--- a/system-files/base/etc/greetd/config.toml
+++ b/system-files/base/etc/greetd/config.toml
@@ -4,3 +4,5 @@ vt = 1
 [default_session]
 command = "dms-greeter --command niri -C /etc/greetd/niri.kdl"
 user = "greeter"
+greeting_message = "Welcome to Bluefin-Wallpapers!"
+language = "en_US.UTF-8"
